### ISSUE TITLE
prometheus-msteams:v1.5.0 pod wont start on openshift kubernetes cluster

### ISSUE THREAD

**Issue by rolfberkenbosch** [prometheus-msteams/prometheus-msteams#208]
Hi,

I have installed the msteams with the helm chart, but i get an error when starting the pod.

```
0s          Warning   Failed                   pod/prometheus-msteams-79cc7cc7b4-mrxfb         Error: container create failed: time="2021-05-18T15:55:58Z" level=error msg="container_linux.go:366: starting container process caused: process_linux.go:472: container init caused: read init-p: connection reset by peer"
```

I am using the image: prometheus-msteams:v1.5.0

I have tried to start the pod in debug mode, but that is also not working and give me the same error.

Any clue how i can debug/solve this ?

**Comment by Knappek** [comment]
Hi @rolfberkenbosch, can you briefly describe the infrastructure you are trying to deploy `prometheus-msteams`? 
This is probably caused by the CRI but no idea what the error actually means.

**Comment by Knappek** [comment]
Oh, I just see in the title that you are running Openshift. It seems like [this issue](https://bugzilla.redhat.com/show_bug.cgi?id=1904558) is related.

**Comment by canelkatmis** [comment]
Increasing limit memory from 32M to 64M fixed it on my cluster(ocp 4.6)
